options/options.go

```diff
@@ -166,6 +167,11 @@ type Options struct {
 	// attempting to probe the build cache. This is only relevant when
 	// GetCachedImage is true.
 	BinaryPath string
+
+	// MagicDir is the path to the directory where all envbuilder files should be
+	// stored. By default, this is set to `/.envbuilder`. This is intentionally
+	// excluded from the CLI options.
+	MagicDir constants.MagicDir
```

Comment:
review: you can now set MagicDir in Options. You can't set it through `ENVBUILDER_MAGIC_DIR` though; this is intentional.